### DISCUSSION TITLE
Equivalence of the as operator to AS keyword

### DISCUSSION THREAD

**Discussion by vxe** [skeeto/emacsql#120] [Support] [answered]
essentially are these possible variations of AS allowed and if so how to express in emacsql, thanks in advance

```
SELECT
  FirstName AS 'First Name',
  LastName AS [Last Name],
  Email AS EMAIL
FROM
  customers;
```

**Comment by tarsius** [comment, marked as answer]
There's an example in the readme at https://github.com/magit/emacsql?tab=readme-ov-file#lists-are-treated-as-expressions.
